swag/restswag.py

```diff
@@ -0,0 +1,141 @@
+from pecan import rest
+import inspect
+
+def decorated_controllers():
+    return []
+
+def collect_controllers(RootController):
```

Comment:
i'd prefer if RootController was root_controller or something similar, mainly because i'd like to reserve CapWords for class names.